Gemfile

```diff
@@ -1,5 +1,7 @@
 source 'https://rubygems.org'
 
+gem 'protected_attributes_continued', '~> 1.4'
```

Comment:
Using this gem looks like a workaround for "not converting the code". Also if we install this it will affect all of Redmine, I would prefer not to do that. Can you update the PR without this gem ?

Also note that the Gemfile is only here for installing mailcatcher, which seems something not suggested by upstream. I am also planning to remove Gemfile.